artemis/modules/api_scanner.py

```diff
@@ -0,0 +1,168 @@
+import json
+import os
+import tempfile
+from typing import Any, Dict, Optional
+
+from karton.core import Task
+from openapi_schema_validator import validate
+from openapi_spec_validator.readers import read_from_filename
+from pydantic import BaseModel
+
+from artemis import http_requests, load_risk_class
+from artemis.binds import Service, TaskStatus, TaskType
+from artemis.config import Config
+from artemis.module_base import ArtemisBase
+from artemis.task_utils import get_target_url
+
+COMMON_SPEC_PATHS = [
+    "/swagger.json",
+    "/v2/swagger.json",
+    "/v3/api-docs",
+    "/openapi.json",
+    "/api-docs",
```

Comment:
add /docs, BTW is there by any chance a longer list? IMO we can use e.g. 50-100 requests to find swagger ;)

Reply:
awesome!